frontend/src/employee-frontend/components/unit/unit-details/UnitEditor.tsx

```diff
@@ -1013,6 +1103,64 @@ export default function UnitEditor(props: Props): JSX.Element {
           onChange={(roundTheClock) => updateForm({ roundTheClock })}
         />
       </FormPart>
+      <FormPart>
+        <div>{showRequired(i18n.unitEditor.label.operationDays)}</div>
+        <FixedSpaceColumn spacing="xs">
+          {([1, 2, 3, 4, 5, 6, 7] as const).map((day) => {
```

Comment:
Using just one source of truth would probably reduce this to `form.operationTimes.map((day, index) => ...`

Reply:
Simplified to produce operation days from times.